src/paperbot/api/routes/sandbox.py

```
@@ -354,3 +362,194 @@ async def get_system_status(http_request: Request):
+    return {}
+
+
+def _save_settings(settings: Dict[str, Any]) -> None:
+    """Save settings to file."""
+    import json
+    SETTINGS_FILE.parent.mkdir(parents=True, exist_ok=True)
+    SETTINGS_FILE.write_text(json.dumps(settings, indent=2))
+
+
+@router.get("/sandbox/settings/e2b")
+async def get_e2b_settings(http_request: Request) -> E2BSettingsResponse:
```

Ruff (0.15.1)

[warning] Unused function argument: `http_request`

(ARG001)